websites/D/discord-botlist/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "MauricePascal#3009",
+      "id": "622784776234991626"
+    },
+    "url": ["discord-botlist.eu", "www.discord-botlist.eu", "status.discord-botlist.eu"],
+    "description": {
+      "en": "Your free Botlist"
```

Comment:
Use this instead: `Your Discord Bot List! We give you the avaibillity to apply your bot and give your bot a page on our website.`